python/src/gz/sim/Model.cc

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2023 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "Model.hh"
+
+namespace py = pybind11;
+
+namespace gz
+{
+namespace sim
+{
+namespace python
+{
+void defineSimModel(py::object module)
+{
+  py::class_<gz::sim::Model>(module, "Model")
+  .def(py::init<gz::sim::Entity>())
```

Comment:
Can you add the copy constructor? See (https://github.com/gazebosim/sdformat/blob/5d6caa2e00b017d1e3ce094699bc2a199d124440/python/src/sdf/pyAirPressure.cc#L39)  and https://github.com/gazebosim/sdformat/blob/5d6caa2e00b017d1e3ce094699bc2a199d124440/python/src/sdf/pyAirPressure.cc#L57-L62

Also, I think we need that for the `World` binding as well. I think it's okay to add it in this PR.

Reply:
Done.